samples/orleans/OrleansAppHost/Program.cs

```diff
@@ -0,0 +1,22 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddAzureProvisioning();
+
+var storage = builder.AddAzureStorage("storage");
```

Comment:
It would be awesome to be able to get this sample working with Azurite. There is a `UseEmulator(...)` extension for this builder which will enable that.

Reply:
It does work already :) I will update the sample to show dev vs non dev